src/org/ice1000/julia/lang/action/julia-repl-actions.kt

```diff
@@ -73,14 +75,74 @@ class JuliaIncludeRunFileToReplAction : JuliaSendCodeToReplAction(
 class JuliaSendSelectionToReplAction : JuliaSendCodeToReplAction(
 	JuliaBundle.message("julia.actions.repl.run-selected.name"),
 	JuliaBundle.message("julia.actions.repl.run-selected.description")), DumbAware {
+
+
 	override fun code(e: AnActionEvent): String? {
-		val editor = e.getData(CommonDataKeys.EDITOR) ?: return null
-		return editor.selectionModel.selectedText
+		val editor = CommonDataKeys.EDITOR.getData(e.dataContext)
```

Comment:
Could you follow the existing style `val editor = something ?: return null` to reduce indentation?

Reply:
Also done here (and in the line below)